sunpy/data/_sample.py

```diff
@@ -127,7 +126,7 @@ def get_sample_file(filename, url_list, show_progress=True, overwrite=False,
                 online_filename += '?raw=true'
             try:
                 url = urljoin(base_url, online_filename)
-                exists = url_exists(url)
+                exists = True
```

Comment:
This is not used in the sample data PR #2993

Reply:
This is now merged.